### DISCUSSION TITLE
curl and wget profiles: shouldnt they be pretty much identical?

### DISCUSSION THREAD

**Discussion by slowpeek** [netblue30/firejail#5342] [Q&A] [answered]
Aside for other things:

|curl|wget|
|-|-|
|private-tmp|#private-tmp|
|protocol inet,inet6|protocol unix,inet,inet6|

So with stock profiles:
- one can download to `/tmp` with wget, but not with curl
- one is allowed to communicate over unix sockets with wget (does wget even support it?), but not with curl

**Comment by ghost** [comment, marked as answer]
Hi, thanks for opening this. At face-value it does make sense that the curl/wget profiles would be pretty much identical. We'll need to dig around in the git history to determine why these observed differences came into existance. As your question and observations do impact the content of these (i would guess widely used) profiles, let's turn this into an 'issue' to attract more attention:

* #5363
